learn/what_is_meilisearch/features.md

```diff
@@ -18,51 +18,48 @@ Meilisearch's default [relevancy rules](/learn/core_concepts/relevancy.md) are d
 
 ## Typo tolerant
 
-Instead of letting typos ruin your search experience, Meilisearch will always find the results you expect.
-Read more about typo tolerance in [this dedicated guide](/learn/configuration/typo_tolerance.md).
+Instead of letting typos ruin your search experience, Meilisearch will always find the results you expect. Read more about typo tolerance in [this dedicated guide](/learn/configuration/typo_tolerance.md).
 
-## Synonyms
-
-![Search demo GIF updating results on the Meilisearch search preview for synonyms 'batman' and 'joker'](/search-synonyms-typo.gif)
-_Meilisearch in action with `batman` and `joker` defined as synonyms_
-
-Defining synonyms lets you craft a more tailored, intuitive search experience.
-Read more about synonyms in [this dedicated guide](/learn/configuration/synonyms.md).
-
-## Highlighting
+## Geosearch
 
-[Highlight](/reference/api/search.md#attributes-to-highlight) query terms so that matches pop out to the eye. Users don't need to read the entire text to find the match.
+Explore the physical world. [Geosearch](/learn/advanced/geosearch.md) allows you to filter and sort results based on their geographic location.
```

Comment:
Interesting, I have the exact same feeling regarding "Search in the real world." I'll go with your suggestion because that's what we currently have on the new landing page.